dashboard/db/migrate/20170502232017_add_course_id_to_plc_course.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseIdToPlcCourse < ActiveRecord::Migration[5.0]
+  def change
+    add_column :plc_courses, :course_id, :integer
```

Comment:
May also want to index this column if we're using it to look up `course.plc_course` ([see `CourseUnit`](https://github.com/code-dot-org/code-dot-org/blob/2e849185932f01c261a731edf8e3939a0138efef/dashboard/app/models/plc/course_unit.rb#L18)).